### ISSUE TITLE
"Generating single mermaid chart"

### ISSUE THREAD

**Issue by janpio** [keonik/prisma-erd-generator#48]
Using the current version, I get the following output:

````
C:\Users\Jan\Documents\throwaway\3130>npx prisma generate
Environment variables loaded from .env
Prisma schema loaded from prisma\schema.prisma
Generating single mermaid chart

✔ Generated Prisma Client (3.7.0 | library) to .\node_modules\@prisma\client in 11.15s 

✔ Generated Entity-relationship-diagram to .\prisma\ERD.svg in 8.82s
You can now start using Prisma Client in your code. Reference: https://pris.ly/d/client
```
import { PrismaClient } from '@prisma/client'
const prisma = new PrismaClient()
```
````

Might be that `Generating single mermaid chart` is some leftover debugging output.

`"prisma-erd-generator": "^0.8.0"`

**Comment by keonik** [comment]
I looked through the repo and couldn't find it anywhere. I have seen the same though. I'll leave it open until a version has been tested without it

**Comment by janpio** [comment]
Then probably something in Mermaid even. Not a big deal fortunately.

**Comment by keonik** [comment]
It looks like they have some logging levels for their config. I could test it out. Or find a way to hide it. I'll take a look

**Comment by keonik** [comment]
Fixed in version `0.9.1`
